### PR TITLE
add page break to omr-generated score skeleton

### PR DESCRIPTION
**Pull request by liang-chen**

### CODE REVIEW

File: omr/importpdf.cpp [hunk 1]
```diff
@@ -165,64 +166,92 @@ void OmrState::importPdfMeasure(OmrMeasure* m, const OmrSystem* omrSystem)
...
       {
-      for (int i = 0; i < omrSystem->measures().size(); ++i) {
-            OmrMeasure* m = &omrSystem->measures()[i];
-            importPdfMeasure(m, omrSystem);
-            }
-      LayoutBreak* b = new LayoutBreak(score);
-      b->setLayoutBreakType(LayoutBreak::Type::LINE);
-      score->lastMeasure()->add(b);
-      return tick;
+          for (int i = 0; i < omrSystem->measures().size(); ++i) {
```
Comment (lasconic) [inline omr/importpdf.cpp]: The indentation is messed up. See https://musescore.org/en/developers-handbook/musescore-coding-rules.
My Xcode settings
capture d ecran 2016-05-25 11 10 24

Comment (liang-chen) [inline omr/importpdf.cpp]: changed the indentations according to musescore code style in omr/importpdf.cpp

File: omr/importpdf.cpp [hunk 2]
```diff
@@ -211,18 +219,37 @@ void OmrState::importPdfPage(OmrPage* omrPage)
...
             LayoutBreak* b = new LayoutBreak(score);
-            b->setLayoutBreakType(LayoutBreak::Type::LINE);
+            b->setLayoutBreakType(LayoutBreak::Type::PAGE);
             measure->add(b);
             }
+          
+      measure = score->firstMeasure();
+      if(measure){
+            MStaff *ms = measure->mstaves()[0];
+            if (!ms->_vspacerUp){
```
Comment (lasconic) [inline omr/importpdf.cpp]: The code style is not respected here too,
if (measure) {

File: omr/importpdf.cpp [hunk 3]
```diff
@@ -211,18 +219,37 @@ void OmrState::importPdfPage(OmrPage* omrPage)
...
+                  Spacer* spacer = new Spacer(score);
+                  spacer->setSpacerType(SpacerType::UP);
+                  spacer->setTrack(0);
+                  measure->add(spacer);
+            }
```
Comment (lasconic) [inline omr/importpdf.cpp]: aligned with the line before.

File: omr/importpdf.cpp [hunk 4]
```diff
@@ -275,8 +303,11 @@ Score::FileError importPdf(MasterScore* score, const QString& path)
 
       OmrState state;
       state.score = score;
-      foreach(OmrPage* omrPage, omr->pages())
-            state.importPdfPage(omrPage);
+          foreach(OmrPage* omrPage, omr->pages()){
```
Comment (lasconic) [inline omr/importpdf.cpp]: indentation is wrong.
You can use `for (OmrPage* ormPage : omr->pages()) {``
Note the space between ) and {

File: omr/omr.cpp [hunk 5]
```diff
@@ -159,7 +159,7 @@ bool Omr::readPdf()
             if(ID != INIT_PAGE && ID != SYSTEM_IDENTIFICATION) {
                   page = 0;
                   progress->setLabelText(QWidget::tr("%1 at Page %2").arg(ActionNames.at(ID+1)).arg(1));
-                  val = omrActions(ID);
+                  val = omrActions(ID,page);
```
Comment (lasconic) [inline omr/omr.cpp]: space after ,

File: omr/omrpage.cpp [hunk 6]
```diff
@@ -1543,73 +1544,28 @@ namespace Ms {
...
-            else {
-                if (found) {
-                    notePeaks.append(Peak(xx1, val, 0));
-                    found = false;
-                }
+        for (int x = x1; x < (x2 - hw); x += step_size) {
+            val = pattern->match(&_page->image(), x, y - hh/2, _page->ratio());
+            if (val >= note_thresh) {
+                notePeaks.append(Peak(x, val, 0));
             }
```
Comment (lasconic) [inline omr/omrpage.cpp]: these are not indented correctly.

File: omr/pattern.cpp [hunk 7]
```diff
@@ -74,14 +74,22 @@ double Pattern::match(const Pattern* a) const
...
-//      int eshift    = (col + w()) & 7;
+          double scr = 0;
+          for (int y = 0; y < rows; ++y) {
+              
+              for(int x = 0; x < cols; x++){
+                  if(col+x >= img->size().width() || row+y >= img->size().height()) continue;
+                  QRgb c = img->pixel(col+x, row+y);
+                  bool black = (qGray(c) < 125);
+                  scr += black?1:0;
+              }
```
Comment (lasconic) [inline omr/pattern.cpp]: indent

**Comment by lasconic** [comment]
I can align the code if you want. Just tell me.

**Comment by liang-chen** [comment]
Aligned many places. Please help do the rest of it if you still observe something not aligned.

**Pull request merged**